templates/_federalist-redirects.njk

```diff
@@ -445,3 +445,11 @@ server {
   server_name govconnect.18f.gov;
   return 301 https://$target_domain;
 }
+
+# login.gov to www.login.gov
+server {
+  listen {{ PORT }};
+  set $target_domain www.login.gov;
+  server_name login.gov;
```

Comment:
```suggestion
  server_name login.gov;
  add_header Strict-Transport-Security 'max-age=31536000; includeSubDomains; preload';
```

Reply:
hsts requirement was previously fulfilled for login.gov by fedralist-proxy server
https://github.com/18F/federalist-proxy/pull/108